main.cpp

```diff
@@ -52,85 +52,95 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: athlete
+//  action 1: an athlete trains in the gym.
+athlete.trains();
+//  action 2: an athlete competes in sporting events.
+athlete.competes();
+//  action 3: an  athlete maintains a healthy lifestyle such as good nutrition and rests.
+athlete.eatsHealthy();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: farmer
+//  action 1: a farmer harvests crops.
+farmer.harvest();
+//  action 2: a farmer raises livestock.
+farmer.raises();
+//  action 3: a farmer manages a farm operations.
+farmer.manages();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Musician
+//  action 1: a musician plays an instrument
+musician.playsInstrument();
+//  action 2: a musician composes music
+musician.composesMusic();
+//  action 3: a musician perfroms in concerts.
+musician.performsConcerts();
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message